src/views/organizations/OrganizationDetailView.vue

```diff
@@ -111,13 +122,17 @@ watchEffect(() => {
     <div v-if="!datasets?.length">
       Pas de jeu de données pour cette organisation.
     </div>
-    <ul v-else class="fr-grid-row fr-grid-row--gutters es__tiles__list">
-      <li v-for="d in datasets" :key="d.id" class="fr-col-12 fr-col-lg-4">
-        <Tile
-          :link="`/datasets/${d.slug}`"
-          :title="d.title"
-          :description="d.description"
-          :img="d.organization?.logo"
+    <ul v-else class="fr-grid-row fr-grid-row--gutters">
+      <li class="fr-col-md-12">
```

Comment:
Il faut mettre le `v-for` sur le `li` 🥵

Reply:
Effectivement, j'étais content que ça ne change pas grand chose au style, maintenant je sais pourquoi 🙈 

En revanche on ne peut pas avoir un espacement vertical entre les cartes cohérent avec http://localhost:5173/datasets sans aller toucher au style de la carte imbriquée (je crois). Est-ce qu'il y a un vrai intérêt à utiliser un ul/li (sachant qu'on utilise donc des div dans la liste des jeux de données) ?

Reply:
Étrange, sur [data.gouv au même endroit](https://www.data.gouv.fr/fr/organizations/ministere-de-linterieur/#/datasets) le `li` prend la taille de la card et la marge collapse.

L'intérêt c'est d'avoir un html sémantique au plus proche de ce qu'on affiche/transmet. Ici on affiche la liste des JDD associés à une org, dans l'idéal représentée par une liste en html.

Reply:
J'ai vraiment pas envie d'aller hacker le style de la carte, sachant qu'il peut changer n'importe quand. Je propose soit de laisser cet espace un peu grand pour l'instant, tant pis, ou de revenir sur une div.

Reply:
Il faudrait passer le `ul` en `display: block` donc en gros lui enlever les classes `fr-grid-row fr-grid-row--gutters`
Tu peux ajouter un `fr-px-0` pour qu'il prenne toute la largeur et c'est bon ! 👍